.gitignore

```
@@ -127,3 +127,4 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+.vscode
```

```suggestion
.vscode

```

please configure your VSCODE to always add an empty line at the end of the file.
ref: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files